buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/IntellijPlugin.kt

```diff
@@ -100,5 +119,8 @@ fun Project.configureIntellijPlugin() {
     password.set(pluginSignPrivateKeyPass)
   }
 
-  tasks.withType<PublishPluginTask> { token.set(jetbrainsMarketplaceToken) }
+  tasks.withType<PublishPluginTask> {
+    dependsOn(verifyChangeLogTask)
```

Comment:
This is a veery late moment to run this task... I'd suggest the following instead:
1. the `prospectiveVersionSection.version != latestVersionSection.version` check can be extracted to a separate tsk and made a dependency of `buildPlugin` task maybe?
2. the check for `prospectiveVersionSection` contents could also be extracted, and then executed from `scripts/enforce-change-notes-updated-on-pr-to-master` (to replace the existing logic there)
3. the existing `verifyChangeLog` could be retained (and depend on both tasks from 1. & 2.)... but only for manual use (not to be run automatically from CI or whatnot)

WDYT?

Reply:
Yeah, that's reasonable 👍